server/utils/referrals/showReferralUtils.test.ts

```diff
@@ -68,6 +68,67 @@ describe('ShowReferralUtils', () => {
     })
   })
 
+  describe('subNavigationItems', () => {
+    const mockReferralId = 'mock-referral-id'
+
+    describe('when viewing the Referral details on the refer journey', () => {
+      it('returns navigation items for the referral pages with the refer paths and sets the Referral details link as active', () => {
+        const currentRequestPath = referPaths.show.personalDetails({ referralId: mockReferralId })
+
+        expect(ShowReferralUtils.subNavigationItems(currentRequestPath, 'referral', mockReferralId)).toEqual([
+          {
+            active: true,
+            href: referPaths.show.personalDetails({ referralId: mockReferralId }),
+            text: 'Referral details',
+          },
+          {
+            active: false,
+            href: referPaths.show.risksAndNeeds.offenceAnalysis({ referralId: mockReferralId }),
+            text: 'Risks and needs',
+          },
+        ])
+      })
+    })
+
+    describe('when viewing the Referral details on the assess journey', () => {
+      it('returns navigation items for the referral pages with the assess paths and sets the Referral details link as active', () => {
+        const currentRequestPath = assessPaths.show.personalDetails({ referralId: mockReferralId })
+
+        expect(ShowReferralUtils.subNavigationItems(currentRequestPath, 'referral', mockReferralId)).toEqual([
+          {
+            active: true,
+            href: assessPaths.show.personalDetails({ referralId: mockReferralId }),
+            text: 'Referral details',
+          },
+          {
+            active: false,
+            href: assessPaths.show.risksAndNeeds.offenceAnalysis({ referralId: mockReferralId }),
+            text: 'Risks and needs',
+          },
+        ])
+      })
+    })
+
+    describe('when viewing the Risks and needs on the refer journey', () => {
+      it('returns navigation items for the risks and needs pages with the refer paths and sets the Risks and needs link as active', () => {
```

Comment:
Probably don't need to test every condition here, but given the `assess > risk and needs` condition isn't tested, I wonder if this and the test above are more broadly `when on the assess journey` and `when on the risks and needs path`?